qt/python/mantidqt/widgets/filefinder/__init__.py

```diff
@@ -9,4 +9,7 @@
 #
 from mantidqt.utils.qt import import_qt
 
-FileFinder = import_qt('..._common', 'mantidqt.widgets.filefinder', 'MWRunFiles')
+FileFinder = import_qt(
```

Comment:
I like the idea of renaming this to be `FileFinderWidget` but as you have said we need to be careful of the API change. Can we deprecate the old with:

```python
FileFinderWidget = ...

def FileFinder(*args, **kwargs):
    from warnings import warn
    warn("The name FileFinder is deprecated, please use FileFinderWidget instead.")
    return FileFinderWidget(*args, **kwargs)
```
?

Reply:
I put back the old `FileFinder` module. It returns a `FileFinderWidget` and log a warning.